packages/kbn-securitysolution-io-ts-list-types/src/request/internal/create_exception_list_schema/index.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as t from 'io-ts';
+
+import {
+  createExceptionListSchema,
+  CreateExceptionListSchemaDecoded,
+} from '../../create_exception_list_schema';
+
+export const internalCreateExceptionListSchema = t.intersection([
+  t.exact(
+    t.type({
+      type: t.keyof({
+        endpoint: null,
+        endpoint_events: null,
+        endpoint_host_isolation_exceptions: null,
+        endpoint_blocklists: null,
+      }),
+    })
+  ),
+  t.exact(
+    t.partial({
+      // TODO: Move the ALL_ENDPOINT_ARTIFACT_LIST_IDS inside the package and use it here instead
+      list_id: t.keyof({
+        endpoint_trusted_apps: null,
```

Comment:
Looks like we're limiting the use of this API to only these `list_id`'s, correct?

if so, could we use the `const` that have the [list id's defined](https://github.com/elastic/kibana/blob/1cbf83f73c5571d329086ee99fc30f62c5e690fc/packages/kbn-securitysolution-list-constants/index.ts#L57) here so that they remain in sync?